rocketpy/EnvironmentAnalysis.py

```diff
@@ -588,22 +594,99 @@ def calculate_maximum_wind_gust(self):
         self.max_wind_gust = np.max(self.wind_gust_list)
         return self.max_wind_gust
 
-    # TODO: Implement
+    # TODO: Create tests
     def calculate_wind_gust_distribution(self):
```

Comment:
`plot_wind_gust_distribution` would be a more appropriate name maybe

Reply:
Completely agreed. Implemented here: [9f33d20](https://github.com/Projeto-Jupiter/RocketPy/pull/166/commits/9f33d20d21891f8a35f3e6cd660b3556c35b735b)